nltk/model/ngram.py

```diff
@@ -11,7 +11,7 @@
 from nltk import compat
 
 
-NEG_INF = -1e6
+NEG_INF = float("-inf")
```

Comment:
I seem to recall that float("-inf") didn't work on some platforms (depends on the underlying version of C). But let's hope that problem is in the past.